packages/sdk/src/localConfig.ts

```diff
@@ -63,7 +63,7 @@ export const localConfig: RawConfig = {
           type: ResourceType.FUNGIBLE,
           assetName: 'USDC',
           assetId: 2000,
-          xsmMultiAssetId: {
+          xcmMultiAssetId: {
```

Comment:
Bit out of scope but can you remove my comments from this file which are a bit random.

Reply:
Done